openchain/peer/handler.go

```diff
@@ -198,6 +200,17 @@ func (d *Handler) beforePeers(e *fsm.Event) {
 
 }
 
+func (d *Handler) beforeBlockAdded(e *fsm.Event) {
+	peerLogger.Debug("Received message: %s", e.Event)
+	msg, ok := e.Args[0].(*pb.OpenchainMessage)
+	if !ok {
+		e.Cancel(fmt.Errorf("Received unexpected message type"))
+		return
+	}
+	// Add the block and any delta state to the ledger
+	_ = msg
```

Comment:
Hi @binhn , why wouldn't this get implemented? if it is needed, maybe I can help.